CHANGELOG.md

```diff
@@ -1,42 +1,22 @@
 # Changelog
 
-## [4.0.0] - 2019-09-04
+## [4.1.0] - 2019-09-24
 
 ### Added
```

Comment:
Should there be an empty "Added"-section?

Reply:
Probably not